src/plugins/intel_gpu/src/graph/primitive_inst.cpp

```diff
@@ -205,8 +216,8 @@ void primitive_inst::update_shape() {
     }
 
     _impl_params->memory_deps = memory_deps;
-    auto new_layouts = _node.type()->calc_output_layouts(_node, *_impl_params);
-    auto new_layout = new_layouts.empty() ? _node.type()->calc_output_layout(_node, *_impl_params) : new_layouts[0];
+
+    auto new_layout = _node.type()->calc_output_layout(_node, *_impl_params);
```

Comment:
Why did you changed this to calc_output_layout instead of calc_output_layouts?

Reply:
Fixed it.